src/index.js

```diff
@@ -0,0 +1,6 @@
+import * as awsCognito from 'amazon-cognito-identity-js';
+import User from './user';
+
+export default {
+  User: new User(awsCognito),
```

Comment:
the naming here is kinda leading me to belive that a "User" is an actual instance of a person that is being logged in, but I get the feeling that its actually like a User "service" where all functions to login the user is consolidated?